mypy.ini

```diff
@@ -1,6 +1,7 @@
 [mypy]
 
 files = torchvision
+python_version = 3.10
```

Comment:
The `type_check_python` job it self runs on Python 3.7 at the moment. Does it make sense to change the version of the job to 3.10 or you think it's totally fine to leave as is?

Reply:
I would keep it 3.7 to make sure contributors can develop with >= 3.7 and are not required to have a higher Python version as base.

Reply:
> Do you know how that works in practice?

I don't, sorry. What I get from this is that will actually run `mypy` with a 3.10 executable if it finds one. In my local setup (and CI as well) it doesn't and so just runs with the regular interpreter. This seems to work out fine. 

Note that you posted the docs for the command line option. The [docs for the configuration file](https://mypy.readthedocs.io/en/stable/config_file.html#confval-python_version) do not mention this behavior.

Reply:
So I assume that if mypy cannot find a 3.10 executable, it won't be able to properly type check code like that of `notusedforsecurity`, unless it comes from a package that is PEP 561 compliant (i.e. it distributes typing stubs)?

I assume that's fine.

But what a F***ing mess :(

Reply:
Did you have a chance to make progress on this @pmeier ? It'd be nice to fix this for those of us who develop on >= 3.8

Reply:
I didn't get a definitive answer, but all my local tests suggest that the wording is just wrong. Even if you have for example a 3.7 *and* 3.10 interpreter on path and `python_version = 3.10` set, if invoked from the 3.7 interpreter, `mypy` will stick with it. Thus, we should be good here.